samples/automl/automlTranslationPredict.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic operations on prediction
+ * with the Google AutoML Translation API.
+ *
+ * For more information, see the documentation at
+ * https://cloud.google.com/translate/automl/docs
+ */
+
+'use strict';
+
+function predict(
+  projectId,
+  computeRegion,
+  modelId,
+  filePath,
+  translationAllowFallback
+) {
+  // [START automl_translation_predict]
+  const automl = require('@google-cloud/automl');
+  const fs = require('fs');
+
+  // Create client for prediction service.
+  const client = new automl.v1beta1.PredictionServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const modelId = `id of the model, e.g. “ICN12345”`;
+  // const filePath = `local text file path of content to be classified, e.g. "./resources/test.txt"`;
+  // const translationAllowFallback = `use Google translation model as fallback, e.g. "False" or "True"';
+
+  // Get the full path of the model.
+  const modelFullId = client.modelPath(projectId, computeRegion, modelId);
+
+  // Read the file content for translation.
+  const content = fs.readFileSync(filePath, 'utf8');
+
+  // Set the payload by giving the content of the file.
+  const payload = {
+    textSnippet: {
+      content: content,
+    },
+  };
+
+  // params is additional domain-specific parameters.
+  // translationAllowFallback allows to use Google translation model.
+  var params = {};

```

Comment:
Do not use `var` - only `const` or `let`.

Apply this comment throughout your PR.